src/Components/AboutControl.jsx

```diff
@@ -69,6 +69,24 @@ function AboutDialog({isDialogDisplayed, setIsDialogDisplayed, installPrefix}) {
  */
 function AboutContent({installPrefix}) {
   const classes = useStyles()
+  const [privacySlider, setPrivacySlider] = useState(0)
```

Comment:
Maybe it's better to have a privacy property that's just the name of the privacy level instead of a boolean.  So Privacy.setPrivacyLevel(Privacy.SOCIAL) and switch on it too:
```
switch(Privacy.getPrivacyLevel()) {
case Privacy.SOCIAL: ...
```